lib/katello/engine.rb

```diff
@@ -166,6 +166,7 @@ class Engine < ::Rails::Engine
       load "#{Katello::Engine.root}/lib/katello/tasks/upgrades/2.1/import_errata.rake"
       load "#{Katello::Engine.root}/lib/katello/tasks/upgrades/2.2/update_gpg_key_urls.rake"
       load "#{Katello::Engine.root}/lib/katello/tasks/upgrades/2.2/update_metadata_expire.rake"
+      load "#{Katello::Engine.root}/lib/katello/tasks/upgrades/2.3/import_package_groups.rake"
```

Comment:
Apologies it looks like all of this may now not make a release until katello 2.4.  mind moving it to a 2.4 directory?